indico/modules/events/editing/client/js/models.js

```diff
@@ -29,6 +29,12 @@ export const EditingReviewAction = {
   requestUpdate: 'request_update',
 };
 
+export const EditableType = {
```

Comment:
We use `Object.keys(EditableTypeTitles)` in two places; can you update those to use your new enum instead?